src/panels/lovelace/views/hui-view-header.ts

```diff
@@ -334,6 +374,21 @@ export class HuiViewHeader extends LitElement {
       display: flex;
     }
 
+    .container:not(.edit-mode) .badges.scroll {
+      overflow: auto;
+      max-width: calc(100% - 16px);
+      scrollbar-color: var(--scrollbar-thumb-color) transparent;
+      scrollbar-width: none;
+      mask-image: linear-gradient(
```

Comment:
`mask-image` is not available in all browsers we support, I think that is fine though in this case, as it will work fine, but look a little less nice. Supported from iOS 15.4 for example.